microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -325,20 +305,23 @@ Enables DWARF support for WebAssembly debugging.  See [Improved WebAssembly debu
 <!-- todo: need text -->
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 115.
-*  This checkbox is present in Microsoft Edge Stable 113.
+*  This checkbox is present in Microsoft Edge Canary 116.
+*  This checkbox is present in Microsoft Edge Stable 114.
 
 
 <!-- ====================================================================== -->
 ## Console: Resolve variable names in expressions using source maps
 
```

Comment:
@captainbrosset @zoherghadyali -- Are there tools in addition to Console that can resolve variable names in expressions using source maps?
In Canary 116, the checkbox label changed from:
Console: Resolve variable names in expressions using source maps
to:
Resolve variable names in expressions using source maps

Reply:
A quick glance at the code makes me think that this isn't only for the Console tool. For example, the Watch sidebar tab in the Sources tool is also probably impacted.

Reply:
"This checkbox affects the **Console**, and the **Watch** sidebar tab in the **Sources** tool."